File: Java/백준/Silver/22993. 서든어택 3/서든어택 3.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(br.readLine());

        Long[] attack = new Long[n];
        StringTokenizer st = new StringTokenizer(br.readLine());
        for (int i = 0; i < n; i++) {
            attack[i] = Long.parseLong(st.nextToken());
        }

        Arrays.sort(attack, 1, n);
        Long a1 = attack[0];

        for (int i = 1; i < n; i++) {
            if (a1 > attack[i]) {
                a1 += attack[i];
            }
            else {
                System.out.println("No");
                System.exit(0);
            }
        }
        System.out.println("Yes");
    }
}

File: Python/백준/Silver/22993. 서든어택 3/서든어택 3.py
import sys
input = sys.stdin.readline

n = int(input())
a = list(map(int, input().split()))

a1 = a[0]
attack = sorted(a[1:])

for player in attack:
    if a1 > player:
        a1 += player
    else:
        print("No")
        break
else:
    print("Yes")